doc/sphinx/source/howto/distribute-train.rst

```diff
@@ -0,0 +1,40 @@
+.. distribute-train.rst 
+
+
+Distribute training 
```

Comment:
typo -> "distributed"

Reply:
@wenzhe-nrv and I discussed a better and more descriptive title to go along with the "How to" theme in this section of the Table of Contents